server-files/index.js

```diff
@@ -13,23 +14,31 @@ const twitter = new twit({
 app.use(express.static(__dirname + './../public'))
 
 app.get('/tweets/:location', (req, res) => {
-  twitter.get('search/tweets', { q: req.params.location, result_type: 'recent', lang: 'en', count: 50 }, (err, data, response) => {
-    if (err) console.log(err)
-    data.statuses.forEach((element, index) => {
-      const twitterSentiment = sentiment(element.text)
+  knexFunc.incrementCount(req.params.location)
+    .then(() => {
+      twitter.get('search/tweets', { q: req.params.location, result_type: 'recent', lang: 'en', count: 50 }, (err, data, response) => {
+        if (err) console.log(err)
+        data.statuses.forEach((element, index) => {
+          const twitterSentiment = sentiment(element.text)
 
-      if (twitterSentiment.score > 0) {
-        element.userSentiment = 'positive'
-      }
-      else if (twitterSentiment.score < 0) {
-        element.userSentiment = 'negative'
-      }
-      else {
-        element.userSentiment = 'neutral'
-      }
+          if (twitterSentiment.score > 0) {
+            element.userSentiment = 'positive'
+          }
+          else if (twitterSentiment.score < 0) {
+            element.userSentiment = 'negative'
+          }
+          else {
+            element.userSentiment = 'neutral'
+          }
+        })
+        res.json(data)
+      })
     })
-    res.json(data)
-  })
 })
 
-app.listen(3000)
+app.get('/database', (req, res) => {
```

Comment:
As mentioned before, `/database` is not a good name. Be more communicative with your naming.